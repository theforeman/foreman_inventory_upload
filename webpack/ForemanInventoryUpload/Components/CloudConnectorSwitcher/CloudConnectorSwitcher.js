import React, { useState } from 'react';
import { translate as __ } from 'foremanReact/common/I18n';
import ShortSwitcherPF4 from '../../../common/Switcher/ShortSwitcherPF4';

const CloudConnectorSwitcher = () => {
  const [cloudConnectorEnabled, setCloudConnectorEnabled] = useState(false);
  return (
    <ShortSwitcherPF4
      id="cloud-connector-settings-switcher"
      label={__('Cloud Connector')}
      isChecked={cloudConnectorEnabled}
      onChange={newIsChecked => setCloudConnectorEnabled(newIsChecked)}
    />
  );
};

export default CloudConnectorSwitcher;
