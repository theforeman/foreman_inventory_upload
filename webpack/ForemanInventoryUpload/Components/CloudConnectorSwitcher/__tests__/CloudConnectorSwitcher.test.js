import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import CloudConnectorSwitcher from '../CloudConnectorSwitcher';

const fixtures = {
  render: {},
};

describe('CloudConnectorSwitcher', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(CloudConnectorSwitcher, fixtures));
});
