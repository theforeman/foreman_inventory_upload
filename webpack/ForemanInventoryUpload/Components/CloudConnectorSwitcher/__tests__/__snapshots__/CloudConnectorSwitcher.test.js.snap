// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloudConnectorSwitcher rendering render 1`] = `
<ShortSwitcherPF4
  id="cloud-connector-settings-switcher"
  isChecked={false}
  label="Cloud Connector"
  onChange={[Function]}
  tooltip={null}
/>
`;
